<launch>
<node name="dt_network" pkg="local_server" type="local_server_node" output="screen">
    <param name="port" value="8080" />
    <param name="ros_pub_topic" value="/from/other"/>
    <param name="ros_sub_topic" value="/to/other"/>
    <param name="data_recv_rate_hz" value="1000"/>
  </node>
</launch>
